deploy_to_kubernetes_cron.sh

```diff
@@ -1,38 +1,40 @@
 #! /bin/bash
-JOB_TEMPLATE="/deploy/kubernetes/cron.job.template.yaml"
-JOB_FILE="/deploy/kubernetes/cron.job.yaml"
 
+# NAME, commands
 source /deploy/kubernetes/commands.sh && get_commands
 source /deploy/kubernetes_deploy_base.sh
 
-JOB_NAME=$NAME
+JOB_TEMPLATE="/deploy/kubernetes/cron.job.template.yaml"
+TEMPORARY_JOB_FILE="/deploy/kubernetes/cron.job.yaml"
 
+# PRIMARY_IMAGE
 /deploy/kubernetes_deploy_base.sh
 
-export PRIMARY_IMAGE="${PRIMARY_IMAGE}"
+# `commands` are an associative array -- [COMMAND]=SCHEDULE
 
+# Get all keys, each as a new string: "${!commands[@]}" 
 for command in "${!commands[@]}"; do
+
+  # Split string into array of strings
   split_command=($command)
-  formatted_command=\"${split_command[0]}\"
```

Comment:
Are we losing some functionality here?

Reply:
Oh right, this needs to be csv. Update in a moment.